web/cgi-bin/missa/propers.pl

```diff
@@ -82,7 +82,9 @@ sub specials {
     if ($item =~ $section_regex) {
       $label = $1;
 
-      if ($rule =~ /omit.*\b$label\b/i) {
+      if ( ($rule =~ /omit.*\b$label\b/i) ||
+           ( ( $version =~ /1570/) && ( $item =~ / Le/ ) ) # omit Leonine prayers issue #367
```

Comment:
1. I know it is poor but title section in translations are out of order, all should be rewritten.
2. Will keep in  mind.